lib/portlayer/network/context.go

```diff
@@ -1073,10 +1086,20 @@ func (c *Context) AddContainer(h *exec.Handle, options *AddContainerOptions) err
 		h.ExecConfig.Networks = make(map[string]*executor.NetworkEndpoint)
 	}
 
+	// If the scope type is external, name the nic as such.  We can only have
```

Comment:
So this is true currently, but I am not sure this will be or should be true in the future. I would just go with a randomly generated name if that is possible.